GPU/Common/DrawEngineCommon.cpp

```diff
@@ -834,7 +837,7 @@ void DrawEngineCommon::SubmitPrim(const void *verts, const void *inds, GEPrimiti
 	dc.verts = verts;
 	dc.inds = inds;
 	dc.vertexCount = vertexCount;
-	dc.indexType = (vertTypeID & GE_VTYPE_IDX_MASK) >> GE_VTYPE_IDX_SHIFT;
+	dc.dec = dec_;
```

Comment:
Hm, we should add `numDrawCalls = 0;` or similar to `NotifyConfigChanged()` in this case, since that can invalidate the decoders.  In theory, that shouldn't happen anyway when there are draw calls, but better safe than crashing...

-[Unknown]